lib/lte_link_control/lte_lc_modem_hooks.c

```diff
@@ -127,3 +127,19 @@ static void on_modem_shutdown(void *ctx)
 		(void)lte_lc_power_off();
 	}
 }
+
+#if CONFIG_UNITY
```

Comment:
We often see `defined` in these `#if` statements:
```suggestion
#if defined(CONFIG_UNITY)
```